app/utils.py

```diff
@@ -27,21 +27,30 @@ def get_info(feature=None, feature_type=None):
     """
     header_dept = [html.H4('Département sélectionné :')]
     header_camera = [html.H4('Caméra sélectionnée :')]
+
     if feature:
+
         if feature_type == 'markers_hover':
             return header_camera + [html.B('Zone: {}'.format(feature['properties']['area']))]
+
         elif feature_type == 'markers_click':
             return header_camera + [html.B(feature['properties']['popup'])]
+
         elif feature_type in ['geojson_alerts', 'geojson_risks']:
             return header_dept + [html.B(feature['properties']['nom'])]
-    # if no object are hovered, it just return standard statement
+
+    # If no object are hovered, it just return standard statement
     return header_dept + [html.P('Faites glisser votre curseur sur un département')]
 
 
 def build_info_object(app_page):
-
+    """
+    This function takes as input the page of interest and outputs the relevant
```

Comment:
As the number of function we have in this script is growing fast, I'm wondering if we should to put set standard in our function's docstrings.
Like this for example: 
1. Brief description of the function
2. Argument (Name + type)
3. Return output

Maybe this would require a specific PR to update it in the whole app.